### ISSUE TITLE
Connecting to different server when attempting to "Continue" on [US-EAST] Uuvana 5 Server

### ISSUE THREAD

**Issue by BalinP** [Uuvana-Studios/longvinter-windows-client#862]
**Describe the bug**
When I started the game I joined the [US-EAST] Uuvana 5 server. I haven't payed on any other server. I played about 4 hours on this server and logged off for the day. The next day I got on, I clicked "Continue ([US-EAST] Uuvana 5)." When I loaded in my character was spawned in the same spot I logged off (Sgt. Lakes Outpost), but I had more money and different gear. I went to where my house was supposed to be and there was a completely different house there. So I logged off and logged back in. No change. I then restarted the game and again there was no change. So I checked the server list for my server and it wasn't there. When I checked recent servers, [US-EAST] Uuvana 5 was listed twice. This has happened a few times, but now when I go to recent servers nothing appears and I can't find it on the normal server list unless I click "Show More." Even then it sometimes won't show. I'm not really sure how to reproduce it cause I can't tell if the problem is on my side or server side, but I'll list everything I did leading up to the bug below. This has also happened to a friend that was in the server too, so I'm not the only one.

**To Reproduce**
1. Install Game and Open
2. Join [US-EAST] Uuvana 5 from the server list
3. Play a little, build a house, remember how much money you have and your inventory
4. Log off and close game
5. Open Game
6. Click "Continue ([US-EAST] Uuvana 5"
7. Check to see if you are in the right server and check your inventory/money

**Expected behavior**
I expected to be put back in the server I had all my stuff on, but instead got put on another server with the same name except I had someone else's gear and my house wasn't there

**Screenshots**
![US EAST Uuvana 5 with my house](https://user-images.githubusercontent.com/104792422/166835272-e8933d80-ddbd-4288-8308-7eeccfdec9e7.png)

![US EAST Uuvana 5 with different house](https://user-images.githubusercontent.com/104792422/166835304-133f3d25-ae05-4a1f-859a-19f6bed24597.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Game Version 1.0.4
 - Steam Version (idk where to find that, but its the most recent one because it updates every I open it)

**Additional context**

**Comment by OGPDTHUMS** [comment]
Yes I lost everything... kind of devastating. I grinded and had two houses fully built on [US-EAST] Uuvana 6 server and now I have nothing. My base had 4 turrets, I had 50k mk, all gone.... Probably had close to 10-15 hours getting a jump when server wiped don't want to play anymore unless it gets fixed or I can get my base back.

**Comment by BalinP** [comment]
the 2 [US-EAST] Uuvana 5 servers seem to have now been merged and someone elses house is clipping into mine
![Servers merged](https://user-images.githubusercontent.com/104792422/167209805-08c35fd5-66f0-4290-be59-a0164bb4134e.png)
.
